pandas/tests/indexing/test_datetime.py

```diff
@@ -123,6 +123,30 @@ def test_consistency_with_tz_aware_scalar(self):
         result = df[0].at[0]
         assert result == expected
 
+    def test_access_datetimeindex_with_timezoned_label(self):
+
```

Comment:
Can you add the github issue number here as a comment?  (see the test after this one for an example)

Reply:
Of course

Reply:
Added now.